CHANGELOG.md

```diff
@@ -3,11 +3,27 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [2.5.5] - 2025-03-06
 ### :bug: Fixed
+- Various reader failover fixes ([PR #1227](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1227)) & ([PR #1246](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1246)).
 - Avoid encoding MariaDB connection properties ([PR #1237](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1237)).
+- Integration test failures ([PR #1251](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1251)) & ([PR #1258](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1258)).
+- Custom domains during failover ([PR #1265](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1265))
+- Skip failover on interrupted thread ([PR #1284](https://github.com/aws/aws-advanced-jdbc-wrapper/pull/1284))
```

Comment:
Lets link to the issues https://github.com/aws/aws-advanced-jdbc-wrapper/issues/1283